crates/rules.bzl

```diff
@@ -53,12 +60,15 @@ def _assemble_crate_impl(ctx):
         "--keywords", ";".join(ctx.attr.keywords),
         "--categories", ";".join(ctx.attr.categories),
         "--description", ctx.attr.description,
-        "--documentation", ctx.attr.documentation,
         "--homepage", ctx.attr.homepage,
         "--license", ctx.attr.license,
         "--repository", ctx.attr.repository,
         "--deps", ";".join(["{}={}".format(k, v) for k, v in ctx.attr.deps.items()]),
     ]
+    if ctx.attr.documentation != "":
```

Comment:
`""` is default value for when a Bazel attribute with type `string` is not assigned